javaHapiValidatorLambda/serverless.yml

```diff
@@ -0,0 +1,35 @@
+#
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+service: fhir-service-validator
+
+provider:
+  name: aws
+  region: us-west-2
+  stage: dev
+  runtime: java11
+
+package:
+  artifact: target/fwoa-hapi-validator-dev.jar
+
+functions:
+  validator:
+    handler: software.amazon.fwoa.Handler
+    timeout: 60
+    memorySize: 2048
+    provisionedConcurrency: 5
+
+resources:
+  - Parameters:
+      Stage:
```

Comment:
Does this `Stage` actually work and change the stage of the stack? i.e. this is how we do it in the main serverless: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/serverless.yaml#L17